src/store/reducers/deploymentsReducer.js

```diff
@@ -159,6 +168,29 @@ const updateDeployedDevicesReducer = (state, { payload: [modules, devices], from
   });
 };
 
+const updateADMDeployedDevicesReducer = (state, { payload, fromAction }) => {
+  const normalizedDevices = normalize(payload, deployedDevicesListSchema).entities.deployedDevices || {};
+  const deployedDevices = Object.keys(normalizedDevices)
+    .reduce(
+      (acc, deviceId) => ({
+        ...acc,
+        [deviceId]: {
+          start: normalizedDevices[deviceId].lastFwUpdateStartTime,
```

Comment:
why not just name these properties like you want them in the model conversion?

Reply:
These are being used in Grid not sent back to the API. These names reflect the column names.